yogi2/src/main/java/frc/robot/ManualInputInterfaces.java

```diff
@@ -166,11 +166,13 @@ private void bindCommandsToDriverXboxButtons()
               this.getGtaInputArcadeDriveX(),
               PowerFactorIncrementDirection.Up),
             new ButtonPress("driverController.kRightBumper", "bumperRight.whenPressed")));
+            /*
```

Comment:
add a comment about why this is commented?

Reply:
agree